### PR TITLE
Milestone5

### PR DESCRIPTION
**Pull request by MarioDena**
Implemented handlers and stateless components for the calculator.

Right now fully functional, even added a way to handle multiple "." and "0" on the input.

Hope you guys find it functional. 

Cheers!

### CODE REVIEW

File: src/logic/Operate.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      break;
+    case '-':
+      result = first.minus(second);
+      break;
+    case 'x':
+      result = first.times(second);
+      break;
+    case '÷':
+      result = first.div(second);
+      break;
```
Comment (shloch) [inline src/logic/Operate.js:19]: handle division by ZERO

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @MarioDena 

Great job...really works great, you joint the components together properly and the project works
calc

I however want you to manage one aspect before you get a thumbs up. I want you to manage division by ZERO. when I want to divide by zero, it's not possible. Make it possible and return `NaN` or `Error`

Make the error none to the user

Thanks

File: src/logic/Operate.js (hunk 1 above)
Comment (MarioDena) [reply src/logic/Operate.js]: On a side note I did Handle the division by 0, that's why the calculator won't let you do something that is impossible. I did what you asked but it is:
1.- Redundant 
2.- Not in the requirements to handle that division in the way you want. Just to handle the problem.

Comment (MarioDena) [reply src/logic/Operate.js]: I did what you asked but this error was already handled in App.js for reference check the code there the behavior was as intended

File: src/logic/Operate.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      result = first.minus(second);
+      break;
+    case 'x':
+      result = first.times(second);
+      break;
+    case '÷':
+      second === 0 ? result = 0 : result = first.div(second);
+      break;
+    case '%':
+      result = first.times(second.div(100));
```
Comment (bellom) [inline src/logic/Operate.js:22]: Awesome work using a ternary operator but what the previous reviewer is saying is; we should have an error message if a user tries to divide a number by `0`.
Something like this; 👇 
image

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @MarioDena  :clap: 

**Great work making `almost` all the necessary corrections from the previous review.**

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/logic/Operate.js (hunk 2 above)
Comment (MarioDena) [reply src/logic/Operate.js:22]: Yes And like I said division by Zero is handled by App.js 
Having an Error message is not specified in the project, thus not mandatory.

**Review by whiz25 — APPROVED** [review]
## Approved :clap: 
You have done well and it's a :+1: from me. Merge and proceed to the next milestone.

### Martin Afani

**Pull request merged**